doc/source/user_guide/configuration.rst

```diff
@@ -43,7 +45,7 @@ Credentials
 To use the SimAI API, your :class:`~ansys.simai.core.client.SimAIClient`
 instance must be authenticated. By default, you are prompted to log in
 via your web browser. However, you can pass your credentials as parameters
-on client creation:
+on client creation in the interactive mode (see `Interactive mode`_.):
```

Comment:
You can also set them in both modes. So I'd just remove that mention or mention it's required in non-interactive mode.

Reply:
True, I misread it.